src/Connectors/test/GemFireConnector.Net4Test/GemFireConnectorFactoryTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is this to fix coverage issues?

Reply:
Testing that code coverage results change... which they didn't ~ something's still wrong with the report and the responses from codecov and sonarcloud are right that the coverage _appears_ to be MIA